src/sentry/services/hybrid_cloud/auth/__init__.py

```diff
@@ -56,6 +56,7 @@ def authentication_request_from(request: Request) -> AuthenticationRequest:
         remote_addr=request.META["REMOTE_ADDR"],
         signature=find_signature(request),
         absolute_url=request.build_absolute_uri(),
+        absolute_url_root=request.build_absolute_uri("/"),
```

Comment:
What's this change for?

Reply:
This change actually just helps some of the `@region_silo(stable=True)` decorated tests I added pass.  It addresses the hybrid cloud support for signed requests (which need the original absolute `/` url of the request to validate).  It is incidental change for the decorators and only effects the hybrid cloud code path.

Reply:
https://github.com/getsentry/sentry/pull/44391/commits/d4f0467c4b09ba3e69bc64b3f4cd9d19859dc95a